indexdata.go

```diff
@@ -102,6 +105,34 @@ type indexData struct {
 	rawConfigMasks []uint8
 }
 
+// readRepoTombstones has to be called after d.repoMetaData has been set.
+func (d *indexData) readRepoTombstones() error {
```

Comment:
it feels like this should be living in `read.go`

Reply:
why? It feels ok to set d.repoTombstones with a method on indexData, doesn't it. Or do you mean just moving the method to read.go?

Reply:
This comment is not relevant anymore. Reading tombstones was replaced by reading metadata.